proto/import_kvpb.proto

```diff
@@ -46,6 +46,10 @@ service ImportKV {
     rpc CleanupEngine(CleanupEngineRequest) returns (CleanupEngineResponse) {}
     // Compact the target cluster for better performance.
     rpc CompactCluster(CompactClusterRequest) returns (CompactClusterResponse) {}
+    // Get current version and commit number
+    rpc GetVersion(GetVersionRequest) returns (GetVersionResponse) {}
+    // Get importer metrics
+    rpc GetMetrics(GetMetricsRequest) returns (GetMetricsResponse) {}
```

Comment:
Dup with `http://pd_or_tikv_addr:port/metrics`?

Reply:
We haven't exposed the StatusServer yet since it depends on TiKvConfig (and IIRC it was in a private module previously; it is now public). 

Even if we do expose the StatusServer there still need to be a way to tell Lightning the `status_port` (which is different from `port`).